rebar.lock

```diff
@@ -132,6 +132,10 @@
   {git,"https://github.com/helium/libp2p-crypto.git",
        {ref,"a969bb7affc7d3fa1472aef80d736c467d3f6d45"}},
   1},
+ {<<"lorawan">>,
+  {git,"https://github.com/helium/erlang-lorawan.git",
+       {ref,"49925a083347e91e74daee3516a2105e82ef6c07"}},
+  1},
```

Comment:
I'd recommend the latest - 32d1c580c26ba41ad5230d080d4c0e23cf273d86
In particular the list_to_existing_atom fixes potential memory issues.